wordpress-theme/skyyrose-flagship/assets/css/product-card-v7.css

```
@@ -0,0 +1,139 @@
+
+.v7card {
+	position: relative;
+	background: #111;
+	border-radius: 14px;
+	overflow: hidden;
+	transition: transform .5s cubic-bezier(.16,.84,.44,1), box-shadow .5s;
+}
+.v7card:hover,
+.v7card:focus-within {
+	transform: translateY(-4px);
+	box-shadow: 0 18px 50px -20px rgba(0,0,0,.8), 0 0 0 1px rgba(var(--v7-accent-rgb),.25);
```

Stylelint (17.13.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)